packages/web-client/app/components/card-pay/layer-two-connect-card/index.ts

```diff
@@ -49,11 +49,11 @@ class CardPayLayerTwoConnectCardComponent extends Component<CardPayLayerTwoConne
   get balancesToShow() {
     return [
       {
-        symbol: 'DAI.CPXD',
+        symbol: this.layer2Network.bridgedDaiTokenSymbol,
```

Comment:
This is for display purposes, which is why I left it. I guess I am OK either way and curious about other people's thoughts.

Reply:
The safe balance card shows `DAI/CARD` on Sokol, not `DAI.CPXD/CARD.CPXD`. I don't mind either, but we should probably be consistent about this.